config/site_settings.yml

```diff
@@ -1406,6 +1406,10 @@ trust:
     default: 0
     client: true
     enum: "TrustLevelSetting"
+  min_trust_level_to_allow_invite:
+    default: 2
+    client: true
```

Comment:
I just saw this is a client setting but doesn't seem to be used on the client side. I think we should remove `client: true` here.

Reply:
@eviltrout said changes are done